### ISSUE TITLE
 [jsx-a11y] A form label must be associated with a control

### ISSUE THREAD

**Issue by khalidelaggan** [cds-snc/report-a-cybercrime#1180]
## Summary

> a11y linter issue 

## Steps to reproduce

With introducing the newly configured a11y linter in the following PR 
https://github.com/cds-snc/report-a-cybercrime/pull/1179/checks?check_run_id=398036525

> CI tests are failing because of the following tests errors

<img width="786" alt="Screen Shot 2020-01-20 at 7 54 56 AM" src="https://user-images.githubusercontent.com/2603117/72728003-56f1bd00-3b5a-11ea-8298-35e2fc49afa5.png">

## Recommended solutions
 
- Review the local test files scopes and criteria
- Ensure that each form label must be associated with a control
- Ensure that each form input must have a valid ID that is linked to the the form label

**Comment by sastels** [comment]
@amazingphilippe @khalidelaggan has this been fixed?

**Comment by amazingphilippe** [comment]
Probably fixed yes. Or in the motions

**Comment by khalidelaggan** [comment]
#1179  is ready to me merged

**Comment by khalidelaggan** [comment]
Once we merge #1179  we can close this issue

**Comment by sastels** [comment]
now fixed by our new checkboxes.
